content/docs/en/elements/components/tab-view.md

```diff
@@ -1,5 +1,44 @@
 ---
 title: TabView
+apiRef: https://docs.nativescript.org/api-reference/modules/_ui_tab_view_
```

Comment:
change it to: https://docs.nativescript.org/api-reference/classes/_ui_tab_view_.tabview

although it makes sense the way you did it, I should probably allow an array because we actually need two links here (I'll do that later, so feel free to leave it as is, will come back and update when ready)